main.tf

```diff
@@ -14,10 +14,29 @@ data "aws_region" "current" {}
 
 data "archive_file" "lambda" {
   type        = "zip"
-  output_path = "${path.module}/lambda.zip"
+  output_path = "./lambda.zip"
```

Comment:
can you explain this change?

Reply:
During my testing I found an issue related to the fact that `apply` know longer creates the archive but instead `plan` when used in a pipeline this was seen to be a major problem as some tools have a limit on how filesare allowed to traverse stages.